Source/SwiftLintFramework/Rules/Style/TrailingNewlineRule.swift

```diff
@@ -61,11 +61,12 @@ public struct TrailingNewlineRule: CorrectableRule, ConfigurationProviderRule, S
         if file.ruleEnabled(violatingRanges: [lastLineRange], for: self).isEmpty {
             return []
         }
-        if count < 1 {
-            file.append("\n")
-        } else {
+
+        if count >= 1 {
```

Comment:
It would be best to rename `count` to `lineCount` or `newLineCount` to avoid triggering the `empty_count` rule